library/core/src/panicking.rs

```diff
@@ -53,6 +53,10 @@ const _: () = assert!(cfg!(panic = "abort"), "panic_immediate_abort requires -C
 #[rustc_do_not_const_check] // hooked by const-eval
 #[rustc_const_stable_indirect] // must follow stable const rules since it is exposed to stable
 pub const fn panic_fmt(fmt: fmt::Arguments<'_>) -> ! {
+    if cfg!(feature = "panic_unreachable_unchecked") {
+        // SAFETY: it's not...
```

Comment:
This safety comment is inadequate, the condition of soundness relies on the programmer asserting panics will never happen, so a more accurate comment is:
```suggestion
        // SAFETY: The user of this flag asserts that a panic will never happen in their codebase
```
  
  
###### if you want to do april fools, go all in :^)